### ISSUE TITLE
Learning rate details

### ISSUE THREAD

**Issue by smitshilu** [yeephycho/tensorflow-face-detection#17]
Can you please share following 3 parameter values for training.

initial_learning_rate
decay_steps
decay_factor

**Comment by yeephycho** [comment]
You can follow the paper and set the parameters as default.
